src/doc/rustc/src/platform-support/fuchsia.md

```diff
@@ -0,0 +1,280 @@
+# `aarch64-fuchsia` and `x86_64-fuchsia`
+
+**Tier: 2**
+
+[Fuchsia] is a modern open source operating system that's simple, secure,
+updatable, and performant.
+
+[Fuchsia]: https://fuchsia.dev/
+
+## Target maintainers
+
+The [Fuchsia team]:
+
+[Fuchsia team]: https://team-api.infra.rust-lang.org/v1/teams/fuchsia.json
```

Comment:
Linking to the teams API is a nice idea. I wonder if we should just do that to avoid getting out of sync. Maybe a note to the effect of "if these differ, consider this link to be authoritative"?

Also, I'd like to say that the preferred way to contact Fuchsia maintainers is to use `@rustbot ping fuchsia` instead of pinging individual members.

Reply:
Updated language to clarify that.